### PR TITLE
allow loading old drm and style files

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
see https://musescore.org/en/node/267856

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
I wonder whether someting like this could/should also be done for style files

File: mscore/editdrumset.cpp [hunk 1]
```diff
@@ -568,8 +568,11 @@ void EditDrumset::load()
       while (e.readNextStartElement()) {
             if (e.name() == "museScore") {
                   if (e.attribute("version") != MSC_VERSION) {
-                        QMessageBox::critical(this, tr("Drumset too old"), tr("MuseScore cannot load this drumset file."));
-                        return;
+                        QMessageBox::StandardButton b = QMessageBox::warning(this, tr("Drumset too old"),
```
Comment (anatoly-os) [inline mscore/editdrumset.cpp]: We discussed these strings somewhere few month ago...
At least, `Drumset too old` is not a correct English sentence afaiu.

Comment (Jojo-Schmitz) [reply mscore/editdrumset.cpp]: `Drumset file too old`?

Comment (Jojo-Schmitz) [reply mscore/editdrumset.cpp]: It is not supposed to be a sentence, but a window title

Comment (Jojo-Schmitz) [reply mscore/editdrumset.cpp]: To what should it get changed then, if at all?

**Pull request merged**